lib/public/domain/enums/detectorsNames.mjs

```diff
@@ -15,7 +15,7 @@
  * Virtual devices introduced for test purpose (TST) or
  * as group of physical detectors in order to present aggregated detectors qualities/statistics (GLO)
  */
-export const VIRTUAL_DETECTOR_NAME = Object.freeze({
+export const NON_PHYSICAL_DETECTOR_NAME = Object.freeze({
```

Comment:
This enum should follow the conventions, so `NonPhysicalDetector`

Reply:
applied